### ISSUE TITLE
crossOrigin not set for FileLoader

### ISSUE THREAD

**Issue by willfalconer** [johh/three-gltf-loader#6]
https://github.com/johh/three-gltf-loader/blob/22c63bf153833c2dd690acb2a28585efd2ebab8e/index.js#L69-L74

As you can see from the above lines, `crossOrigin`/`withCredentials` is not set for FileLoader, however, it is set for GltfParser and TextureLoader.

FileLoader has a different interface compared to GltfParse and TextureLoader. The change required is slightly different. Something along the lines of:

```
if (crossOrigin === 'use-credentials') loader.setWithCredentials(true);
```

Currently, cookie-based authentication cannot be used with GltfLoader.

**Comment by willfalconer** [comment]
Issue also raised in ThreeJS https://github.com/mrdoob/three.js/issues/16600

**Comment by johh** [comment]
This should be fixed by mrdoob/three.js#16601. Once it is merged and released, the changes will be mirrored by `three-gltf-loader`.

**Comment by johh** [comment]
The related fix is included in 1.106.0.
